integration/tests/cook/test_cli.py

```diff
@@ -12,6 +12,7 @@
 from urllib.parse import urlparse
 
 from nbconvert.preprocessors import ExecutePreprocessor
+from retrying import retry
```

Comment:
Nitpick: It's annoying that we use `tenacity` in the CLI but `retrying` in the integration tests. We should just switch to `tenacity` everywhere for consistency.